content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -41,11 +49,23 @@ def self.get_runners_count_by_ems(ems, factory_config)
               transformation_hosts(ems, factory_config).inject(0) { |sum, thost| sum + thost[:runners][:current] }
             end
 
+            def self.ems_max_runners(ems, factory_config, max_runners = DEFAULT_EMS_MAX_RUNNERS, handle = $evm)
+              if ems.custom_get('Max Transformation Runners')
+                handle.log(:info, "Using max transformation runners from EMS custom attribute: #{ems.custom_get('Max Transformation Runners')}")
+                ems.custom_get('Max Transformation Runners').to_i
+              elsif factory_config['ems_max_runners']
+                handle.log(:info, "Using max transformation runners from factory config: #{factory_config['ems_max_runners']}")
+                factory_config['ems_max_runners'].to_i
+              else
+                handle.log(:info, "Using default max transformation runners: #{max_runners}")
+                max_runners
+              end
+            end
+
             def self.get_transformation_host(task, factory_config, handle = $evm)
               ems = handle.vmdb(:ext_management_system).find_by(:id => task.get_option(:destination_ems_id))
-              ems_max_runners = ems.custom_get('Max Transformation Runners').to_i || factory_config['ems_max_runners'] || 1
               ems_cur_runners = get_runners_count_by_ems(ems, factory_config)
-              transformation_host_hash = ems_cur_runners < ems_max_runners ? eligible_transformation_hosts(ems, factory_config).first : {}
+              transformation_host_hash = ems_cur_runners < ems_max_runners(ems, factory_config) ? eligible_transformation_hosts(ems, factory_config).first : {}
```

Comment:
Were you planning to change this call to `ems_max_runners` to pass `1` as the third parameter to override `max_runners`?  That would maintain the previous logic.

Reply:
Nope. It was `1` just because I didn't know what could be a reasonable value. Now that RHV perf team has said that 10 should be the default per host, I'm setting it to 10 at the EMS level. This would allow 1 conversion host per EMS by default. Then user can override it using custom attribute.